onnxruntime/test/mlas/unittest.cpp

```diff
@@ -2495,6 +2495,206 @@ class MlasComputeExpTest : public MlasTestBase
     }
 };
 
+template <typename T>
+class MlasQLinearAddExpTest : public MlasTestBase
```

Comment:
The "Exp" here is from the MlasComputeExpTest that was testing MlasComputeExp. "Exp" isn't needed here.

Reply:
removed